packages/cerebral-provider-firebase/src/helpers.js

```diff
@@ -110,3 +110,7 @@ export function convertObjectWithTemplates (obj, resolve) {
     return convertedObject
   }, {})
 }
+
+export function noop () {
+  return function () {}
+}
```

Comment:
oh, fudge... I was a bit quick here, should be just plain function yes... working on something else now, will fix it tomorrow as we need a final test anyways (running all the code against large codebase)